### PR TITLE
coverity 1021749: Uninitialized pointer field

### PR DESCRIPTION
**Pull request by bryancall**

### CODE REVIEW

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/467/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/476/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/463/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2159/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2055/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/590/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/719/

File: iocore/eventsystem/UnixEThread.cc [hunk 1]
```diff
@@ -42,26 +42,13 @@ struct AIOCallback;
...
 
 EThread::EThread()
-  : generator((uint64_t)Thread::get_hrtime_updated() ^ (uint64_t)(uintptr_t)this),
-    ethreads_to_be_signalled(nullptr),
-    n_ethreads_to_be_signalled(0),
-    id(NO_ETHREAD_ID),
-    event_types(0),
-    signal_hook(nullptr),
-    tt(REGULAR)
+  : generator((uint64_t)Thread::get_hrtime_updated() ^ (uint64_t)(uintptr_t)this), id(NO_ETHREAD_ID), signal_hook(nullptr)
```
Comment (PSUdaemon) [inline iocore/eventsystem/UnixEThread.cc]: You can factor out the `signal_hook` init too.

File: iocore/cache/RamCacheLRU.cc [hunk 2]
```diff
@@ -36,8 +36,8 @@ struct RamCacheLRUEntry {
 
 struct RamCacheLRU : public RamCache {
   int64_t max_bytes = 0;
-  int64_t bytes = 0;
-  int64_t objects = 0;
+  int64_t bytes     = 0;
+  int64_t objects   = 0;
```
Comment (PSUdaemon) [inline iocore/cache/RamCacheLRU.cc]: rebase to remove these changes.

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/486/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/495/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/483/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2178/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/609/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2074/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/738/

**Pull request merged**